src/apps/testapps/testCompactCells.c

```diff
@@ -297,24 +295,23 @@ SUITE(compactCells) {
         }
         uncompactCellsResult = H3_EXPORT(uncompactCells)(
             someHexagons, numHex, uncompressed, numHex * 7, MAX_H3_RES + 1);
-        t_assert(uncompactCellsResult != 0,
+        t_assert(uncompactCellsResult == E_FAILED,
                  "uncompactCells fails when given resolutions beyond max");
     }
 
     TEST(someHexagon) {
         H3Index origin;
         setH3Index(&origin, 1, 5, 0);
 
-        int64_t childrenSz = H3_EXPORT(uncompactCellsSize)(&origin, 1, 2);
+        int64_t childrenSz;
+        t_assertSuccess(
+            H3_EXPORT(uncompactCellsSize)(&origin, 1, 2, &childrenSz));
```

Comment:
This is rouhgly my usage. There may be some cases where I use `t_assertSuccess` for logic under test, but usually not so as to preserve the message. I think it can be treated as just a convenient way to write the `t_assert` invocation, as the messages are fairly opaque anyways (unless the message is used to clarify what is going on in the test)